### DISCUSSION TITLE
Vercel deploying error  "ELIFECYCLE  Command failed with exit code 128. Error: Command "pnpm install" exited with 1"

### DISCUSSION THREAD

**Discussion by MikeSporris300** [vercel/community#3153] [Help]
### Summary

I keep getting this error after creating and deploying the repo from terminal on vs code to Vercel

ELIFECYCLE  Command failed with exit code 128.
--
Error: Command "pnpm install" exited with 1

### Steps to Reproduce

<img width="1317" alt="Screenshot 2023-07-07 at 1 07 17 AM" src="https://github.com/vercel/community/assets/120545342/bc4f0e03-e45d-47d1-a708-e878c5c041d7">

**Comment by mattquestions** [comment]
Has anyone found the fix?

**Reply by ramnes** [reply]
@umitmertcakmak see below

**Comment by ramnes** [comment]
In our case, the problem seems related to the build cache, not sure why.

We fixed it with `VERCEL_FORCE_NO_BUILD_CACHE=1` in the project environment.

**Comment by eivindml** [comment]
Had the same experience today. Suddenly got this error I've never gotten before. Setting the ENV mentioned above fixed it.

**Comment by kingsmen732** [comment]
VERCEL_FORCE_NO_BUILD_CACHE=1  doesnt work now
